common-custom-user-data-gradle-plugin/src/main/java/com/gradle/Utils.java

```diff
@@ -105,6 +106,19 @@ static String urlEncode(String str) {
         }
     }
 
+    /**
+     * Redact any `user:password` that is part of the URL.
+     */
+    static String redactUserInfo(String url) {
```

Comment:
I would remove the javadoc. IMO, the method name says it all. (for Gradle and Maven)

Reply:
Done